python/az/aro/azext_aro/_help.py

```diff
@@ -31,6 +31,16 @@
     text: az aro list -o table
 """
 
+helps['aro list-install-versions'] = """
```

Comment:
can we change the command to something like `az aro get-versions`?

Reply:
+1, it'll align [with AKS' command](https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#azure-portal-and-cli-versions).  We should also run this by Jerome to make sure he's good with it.

Reply:
I have updated the command to align with AKS'.